frontend/package.json

```diff
@@ -127,8 +127,8 @@
     "url": "npx playwright install chromium && npx playwright test --project=URL"
   },
   "engines": {
-    "node": ">=19",
-    "npm": ">=6.14.8"
+    "node": ">=20.11.0",
+    "npm": ">=10.2.4"
```

Comment:
![image](https://github.com/SatcherInstitute/health-equity-tracker/assets/20135478/ee3a52e5-3942-460d-8cb9-e51906a88635)

https://nodejs.org/en/about/previous-releases#looking-for-latest-release-of-a-version-branch